README.adoc

```diff
@@ -285,6 +285,104 @@ spec:
 ...
 ----
 
+== Configure Health
+
+If you want customize the health alerts in Kiali you can do it in the configuration yaml.
+
+=== Kiali defaults
+
+By default Kiali health is:
+
+* Degraded when Error Rate is >= 0.1%
+* Failure when Error Rate is >= 20%
+
+For Kiali an error request is when:
+
+* For http the code status match the expression */^[4-5]\d\d$/*
+* For grpc the code status match the expression */^ [1-9]$|^1[0-6]$/*
+
+=== Configuration
+
+Kiali works with a FIFO configuration, so Kiali will use the first configuration that applies with the resource requested.
+
+So the granularity and criteria should be configured from highest to lowest.
+
+The health configuration follow this scheme:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: string
+      kind: string
+      name: string
+      tolerance:
+        - request_protocol: string
+          request_type: string
+          code: string
+          degraded: number
+          failure: number
+          ...
+    ...
+
+
+----
+
+For each Rate Kiali will use the first match by:
+
+* Namespace (string) is a regular expression that match the namespace name (bookinfo, istio-system...)
+* Kind (Optional) (string) is a regular expression that match the resource type (app/workload/service)
+* Name (Optional) (string) is a regular expression that match the resource name (reviews, productpage...)
+
+If a configuration match kiali will use the first tolerance configuration that match:
+
+* Request protocol is a plain string (http/grpc...)
+* Request type (string) is a regular expression that match the request (inbound/outbound)
```

Comment:
I would prefer `direction` than `request_type`, it's more meaningful.

Reply:
Also, I don't think it's worth it to run regexp checks to match these strings when there's only 2 or 3 possible values. It's not the same cost as a direct string match, so in this case it sounds like a an overkill. As for "Kind" above.

Reply:
mmm I was thinking about ...the user want to have the health for inbound and outbound...or in kind case for wokload and app...

Reply:
But if you don't set these settings (driection or kind), I guess it stands for "all", no?
Unset direction means both inbound and outbound ; unset kind means any kind of entity?

The only missing possibility would be if you want to apply something, for instance, only on a app and service but not the related workloads ... but it seems really like an unusual corner case and anyway it's still possible to do by adding more rate configs, no?